### PR TITLE
Add integration tests for changes to resources.yaml

### PR DESCRIPTION
**Pull request by tmckayus**
This change adds travis tests which check whether or not
resources.yaml in the current pull request or branch differs
from resources.yaml on the master branch. If resources.yaml
is different, the template integration tests from the oshinko-s2i
repository are run against the templates created by resources.yaml.
If resources.yaml is not different, the test returns success.

### CODE REVIEW

**Comment by tmckayus** [comment]
Note that the first commit has the tests turned on unconditionally so that we can prove they run and pass.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+sudo: required
+dist: trusty
+language: go
+## home folder is /home/travis/gopath/src/github.com/radanalyticsio/oshinko-s2i
```
Comment (tmckayus) [inline .travis.yml]: wrong comment, fix when conditional exit is restored

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+services:
+- docker
+matrix:
+  # no need to include the build target at present because the template
```
Comment (tmckayus) [inline .travis.yml]: doesn't apply, cut and pasted from s2i

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      sudo ls -l $ORIGIN_DIR
+      sudo tar -C $ORIGIN_DIR -xvzf $ORIGIN_DIR/openshift-origin-client-tools-v1.5.1-7b451fc-linux-64bit.tar.gz
+      sudo tar -C $ORIGIN_DIR -xvzf $ORIGIN_DIR/openshift-origin-server-v1.5.1-7b451fc-linux-64bit.tar.gz
+      sudo cp $ORIGIN_DIR/openshift-origin-client-tools-v1.5.1-7b451fc-linux-64bit/oc $ORIGIN_DIR
+      sudo cp $ORIGIN_DIR/openshift-origin-server-v1.5.1-7b451fc-linux-64bit/* $ORIGIN_DIR
+      sudo chmod -R +755 $ORIGIN_DIR/*
+      sudo cp $ORIGIN_DIR/* /bin
+      sudo ls -l /bin
+      oc version
+      ## export PATH=$PATH:$ORIGIN_DIR
```
Comment (tmckayus) [inline .travis.yml]: path change can go

**Comment by elmiko** [comment]
this lgtm, thanks Trevor!

**Pull request merged**